transcriptTransformationScript.js

```diff
@@ -35,6 +35,82 @@ function addSeparationAttribute(currEntry) {
+                            for (var item of bodyItem["items"]) {
+                                if (item.hasOwnProperty("columns")) {
+                                    convertColumnsWidthToString(item);
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+// Handler 4
+function convertColumnAttributesToCamelCase(currEntry) {
+    if (currEntry['type'] === 'message') {
+        if (currEntry.hasOwnProperty("attachments")) {
+            const attachments = currEntry["attachments"];
+            attachments.forEach(attachment => {
+                if (attachment.hasOwnProperty("content")) {
+                    const content = attachment["content"];
+                    if (content.hasOwnProperty("body")) {
+                        const contentBody = content["body"][0];
+                        const bodyItems = contentBody["items"];
+                        bodyItems.forEach(bodyItem => {
+                            if (bodyItem.hasOwnProperty("columns")) {
+                                const columns = bodyItem["columns"];
+                                columns.forEach(column => {
+                                    if (column.hasOwnProperty("items")) {
+                                        const colItems = column["items"];
+                                        colItems.forEach(colItem => {
+                                            if (colItem.hasOwnProperty("size")) {
+                                                colItem["size"] = colItem["size"].charAt(0).toLowerCase() + colItem["size"].slice(1, colItem["size"].length);
+                                            }
+                                            if (colItem.hasOwnProperty("weight")) {
+                                                colItem["weight"] = colItem["weight"].charAt(0).toLowerCase() + colItem["weight"].slice(1, colItem["weight"].length);
+                                            }
+                                            if (colItem.hasOwnProperty("color")) {
+                                                colItem["color"] = colItem["color"].charAt(0).toLowerCase() + colItem["color"].slice(1, colItem["color"].length);
+                                            }
+                                            if (colItem.hasOwnProperty("horizontalAlignment")) {
+                                                colItem["horizontalAlignment"] = colItem["horizontalAlignment"].charAt(0).toLowerCase() + colItem["horizontalAlignment"].slice(1, colItem["horizontalAlignment"].length);
+                                            }
+                                            if (colItem.hasOwnProperty("spacing")) {
+                                                colItem["spacing"] = colItem["spacing"].charAt(0).toLowerCase() + colItem["spacing"].slice(1, colItem["spacing"].length);
```

Comment:
To avoid this long code, you can define `const attributesToEdit = ["size", "weight", "color", "horizontalAlignment", "spacing"]` and iterate over it.

Reply:
Yes, I thought about it, but in some places maybe it is expected to be uppercase at the beginning because of that I followed this long route.

Reply:
Won't it be equivalent to your code?

Reply:
No, the code changes some specific fields
if it iterates over the whole object; it may change some attributes that supposed to have an uppercase at the beginning, so such a change will cause the tests to fails
because of that, i am just changing these specific fields

Reply:
Maybe I wasn't clear, I meant something like that:
```
const attributesToEdit = ["size", "weight", "color", "horizontalAlignment", "spacing"];
.
.
colItems.forEach(colItem => {
     for (const att of attributesToEdit) {
            if (colItem.hasOwnProperty(att)) {
                colItem[att] = colItem[att].charAt(0).toLowerCase() + colItem[att].slice(1, colItem[att].length);
             }
     }
}
```

Reply:
Oh sorry i got you in a wrong way 🙄
You are right, this would be much better 👍, will change it